README.md

```diff
@@ -479,6 +479,38 @@ Returns the [`PlacesService`](google.maps.places.PlacesService) class to use dir
 google.maps.places.PlacesService
 ```
 
+### useMaxZoomService
+
+React hook to use the [Maximum Zoom Imagery Service](https://developers.google.com/maps/documentation/javascript/maxzoom) in any component.
+
+#### Usage
+
+```jsx
+import React from 'react';
+import {useMaxZoomService} from '@ubilabs/google-maps-react-hooks';
+
+const MyComponent = () => {
+  const maxZoomService = useMaxZoomService();
+
+  maxZoomService.getMaxZoomAtLatLng(
+    LatLng,
+    (result: google.maps.MaxZoomResult) => {
+      // Do something with result
+    }
+  );
```

Comment:
Maybe this makes it clearer, that a `location` is passed and what it is.
```suggestion
```tsx
import React, {useEffect} from 'react';
import {useMaxZoomService} from '@ubilabs/google-maps-react-hooks';

const MyComponent = () => {
  const maxZoomService = useMaxZoomService();
  const location = /** google.maps.LatLng */;

  useEffect(() => {
    maxZoomService?.getMaxZoomAtLatLng(
      location,
      (result: google.maps.MaxZoomResult) => {
        // Do something with result
      }
    );
  }, [location]);
```